proto/ibc/applications/transfer/v1/tx.proto

```diff
@@ -25,6 +26,7 @@ service Msg {
 // ICS20 enabled chains. See ICS Spec here:
 // https://github.com/cosmos/ibc/tree/master/spec/app/ics-020-fungible-token-transfer#data-structures
 message MsgTransfer {
+  option (amino.name)           = "cosmos-sdk/MsgTransfer";
```

Comment:
As far as I understand, these names are for backwards compatibility and must match that which is used in `RegisterConcrete` legacy amino cdc. 

I'm unsure if any other annotations are required here.

Reply:
here is a good example: https://github.com/cosmos/cosmos-sdk/blob/44e90e061effe955ad52d4b191897bdff41426a8/proto/cosmos/bank/v1beta1/tx.proto#L38-L54


i am working on adding docs on this but bogged down with many things. 


on coin you may need to add the extra notations. Id recommmend some golden tests to make sure they arent breaking. If they are breaking then it could break some frontends based on how they are written

Reply:
Thank you! docs would be great, but no rush on that! This is great and I can try to do some testing with a ledger. 

I saw this usage of `legacy_coins` alright, but I wasn't sure where this is inferred somewhere in the go code? Is `legacy_coins` somewhere else, which ties things together? I think this will be needed for coins on `MsgTransfer`, will add that and test things out!